### PR TITLE
Rewrite CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kit-ty-kate**
This PR rewrites the documentation at destination to package maintainers in hopes of being more helpful for beginner and well as experimented publishers.

This work, as well as the associated changes to the [opam-repository wiki](https://github.com/ocaml/opam-repository/wiki) was part of an effort ongoing since August last year for myself to be able to retire from the maintenance of opam-repository as smoothly as possible, after 6 and half years spent maintaining it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,263 +5,185 @@ Contributions under the form of new packages, issue reports and pull
...
-The .opam file can then be generated using the following command:
+Fixing packages
+---------------
 
-```
-$ dune-release opam pkg
-[-] Creating opam package description for hello
-[+] Wrote opam package description _build/hello.0.0.1/opam
-```
+Packages are fixed as they show up as broken, if the opam-repository maintainers have time.
```
Comment (mseri) [inline CONTRIBUTING.md:142]:
```suggestion
Packages are fixed as soon as they show up as broken in opam-repository CI or in the repository issues, if the opam-repository maintainers have time.
```

**Comment by mseri** [comment]
I only have a minor suggestion. Thanks a lot!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,263 +5,185 @@ Contributions under the form of new packages, issue reports and pull
...
-[?] Upload _build/hello-v0.0.1.tbz as release asset? [Y/n]
-Y
-[-] Uploading _build/hello-v0.0.1.tbz as a release asset for v0.0.1 via github's API
+git switch -c release-yourpkg-version
+git add packages/yourpkg/yourpkg.version/
+git commit -pm "Release yourpkg.version"
+git remote add your-github-handle git@github.com:your-github-handle/opam-repository.git
+git push your-github-handle release-yourpkg-version
 ```
+and open a PR on opam-repository on GitHub. Congratulation!
```
Comment (jmid) [inline CONTRIBUTING.md:135]: Nit: `Congratulations!`  (missing `s`)

Comment (kit-ty-kate) [reply CONTRIBUTING.md:135]:
```suggestion
and open a PR on opam-repository on GitHub. Congratulations!
```

**Comment by mseri** [comment]
Thanks a lot!

**Pull request merged**